tequilapi/endpoints/connection.go

```diff
@@ -56,6 +56,26 @@ type statusResponse struct {
 	SessionID string `json:"sessionId,omitempty"`
 }
 
+// swagger:model
+type ipResponse struct {
```

Comment:
Very strange that structures are private.
Lets make them public, because it is so.